reo/tests/test_reopt_url.py

```diff
@@ -214,8 +265,8 @@ def test_complex_incentives(self):
         c = nested_to_flat(d['outputs'])
 
         d_expected = dict()
-        d_expected['lcc'] = 10959381
-        d_expected['npv'] = 11246374 - d_expected['lcc']
+        d_expected['lcc'] = 10972574
```

Comment:
were these values changed to align with Julia model results, or are they from the private branch? If they're from the private branch then I don't think that they should be changed